app/src/org/commcare/activities/connect/ConnectIdActivity.java

```diff
@@ -0,0 +1,166 @@
+package org.commcare.activities.connect;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+
+import org.commcare.android.database.connect.models.ConnectUserRecord;
+import org.commcare.connect.ConnectConstants;
+import org.commcare.connect.ConnectDatabaseHelper;
+import org.commcare.connect.ConnectManager;
+import org.commcare.dalvik.R;
+import org.commcare.fragments.connectId.ConnectIdBiometricConfigFragment;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.NavController;
+import androidx.navigation.NavDirections;
+import androidx.navigation.fragment.NavHostFragment;
+
+
+public class ConnectIdActivity extends AppCompatActivity {
+    static NavController controller;
+    static NavDirections navDirections = null;
```

Comment:
This can be changed to a local variable in each of the functions where it's used, no need to hang on to it long-term